cellmerged.go

```diff
@@ -33,6 +33,24 @@ func (f *File) GetMergeCells(sheet string) ([]MergeCell, error) {
 	return mergeCells, err
 }
 
+// UnmergeCell unmerge a MergeCell.
+func (f *File) UnmergeCell(sheet string, m MergeCell) error {
```

Comment:
After further thinking, I propose that the `UnmergeCell` and [`MergeCell`](https://godoc.org/github.com/360EntSecGroup-Skylar/excelize#File.MergeCell) functions use the same formal parameters to maintain semantics. The [`MergeCell`](https://godoc.org/github.com/360EntSecGroup-Skylar/excelize#MergeCell) structure just used to get merged cells.

Reply:
Ok. I didn't find the MergeCell function in cell.go. Maybe the `MergeCell` will be moved into cellmerged.go in the future or move `UnmergeCell` to cell.go?

Reply:
Yeah right, the `MergeCell` should be moved into `cellmerged.go`.